src/webgpu/api/validation/createTexture.spec.ts

```diff
@@ -71,58 +68,149 @@ class F extends ValidationTest {
 
 export const g = makeTestGroup(F);
 
-g.test('mipLevelCount_validation_on_formats')
-  .params(poptions('format', kAllTextureFormats))
+g.test('zero_size')
+  .desc(
+    `Test texture creation with zero or nonzero size of
+   width, height, depthOrArrayLayers and mipLevelCount for every dimension, and representative formats.`
+  )
+  .params(
+    params()
+      .combine(poptions('dimension', ['1d', '2d', '3d'] as const))
+      .combine(
+        poptions('zeroArgument', [
+          'none',
+          'width',
+          'height',
+          'depthOrArrayLayers',
+          'mipLevelCount',
+        ] as const)
+      )
+      .combine(
+        poptions('format', [
+          'rgba8unorm',
+          'rgb10a2unorm',
+          'depth24plus-stencil8',
+          'bc1-rgba-unorm',
```

Comment:
1d/3d compressed textures are an unknown, I don't know what should happen here.

Reply:
We disallow 1D/3D compressed texture in Dawn project. I suppose it is correct. So I added a filter for compressed format.